packages/nouns-contracts/contracts/NounsToken.sol

```diff
@@ -105,13 +114,16 @@ contract NounsToken is INounsToken, Ownable, ERC721Checkpointable {
         address _minter,
         INounsDescriptorMinimal _descriptor,
         INounsSeeder _seeder,
-        IProxyRegistry _proxyRegistry
+        IProxyRegistry _proxyRegistry,
+        bytes32 merkleroot
```

Comment:
nit pick - put an underscore for merkleroot to align with other input param format